package.json

```diff
@@ -90,6 +91,7 @@
     "stylelint-config-prettier": "^3.2.0",
     "stylelint-scss": "^3.0.0",
     "stylelint-webpack-plugin": "^0.10.5",
+    "svg-spritemap-webpack-plugin": "^2.6.0",
```

Comment:
also uninstall "iconfont-plugin-webpack" if it's no longer needed for our iconfonts

Reply:
Good catch. Removed in c1cdb7f